frontend/js/tests/user/stats/UserArtistEvolutionActivity.test.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from "react";
+import { screen, waitFor, act } from "@testing-library/react";
+import { SetupServerApi, setupServer } from "msw/node";
+import { http, HttpResponse } from "msw";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import ArtistEvolutionActivityStreamGraph, {
+  UserArtistEvolutionActivityProps,
+} from "../../../src/user/stats/components/UserArtistEvolutionActivity";
+import * as userArtistEvolutionActivityResponse from "../../__mocks__/userArtistEvolutionActivity.json";
+import { renderWithProviders } from "../../test-utils/rtl-test-utils";
+
+const userProps: UserArtistEvolutionActivityProps = {
+  user: { name: "foobar" },
+  range: "week",
+};
+
+jest.mock("@nivo/stream", () => ({
+  ...jest.requireActual("@nivo/stream"),
+  ResponsiveStream: ({ children }: any) =>
+    children ? children({ width: 400, height: 400 }) : <div>Mock Stream Chart</div>,
+}));
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: { retry: false, staleTime: 0, gcTime: 0 },
+  },
+});
+
+const reactQueryWrapper = ({ children }: any) => (
+  <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
+);
```

Comment:
You can import both of these from a utility file: 
https://github.com/metabrainz/listenbrainz-server/blob/master/frontend/js/tests/test-react-query.tsx